common/commands/config_test.go

```diff
@@ -344,6 +344,67 @@ func TestImport(t *testing.T) {
 	assert.Equal(t, "password", serverDetails.GetPassword())
 }
 
+func TestCommandName(t *testing.T) {
+	cc := NewConfigCommand(AddOrEdit, testServerId)
+
+	// Test when the environment variable is not set
+	err := os.Unsetenv(coreutils.UsageOidcConfigured)
+	assert.NoError(t, err)
+	assert.Equal(t, ConfigCommandName, cc.CommandName())
+
+	// Test when the environment variable is set
+	err = os.Setenv(coreutils.UsageOidcConfigured, "true")
+	assert.NoError(t, err)
+
+	assert.Equal(t, ConfigOidcCommandName, cc.CommandName())
+
+	// Clean up
+	err = os.Unsetenv(coreutils.UsageOidcConfigured)
+	assert.NoError(t, err)
+}
+
+func TestConfigCommand_ExecAndReportUsage(t *testing.T) {
```

Comment:
Let's add a defer function at the beginning of this function, that unsets the `UsageOidcConfigured` variable.